meta/autoid/autoid.go

```diff
@@ -29,6 +30,12 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	minStep            = 5000
```

Comment:
Can we make minStep lower like 1000?

Reply:
yes

Reply:
1000 is too small, let's say we want 200000 insert QPS in a single tidb server, how many transaction do we need? what if we met transaction conflict while updating the allocated meta.

Reply:
The `minStep` only matters when the insert QPS is very low.